latest/ug/automode/auto-accelerated.adoc

```diff
@@ -0,0 +1,342 @@
+[.topic]
+[#auto-accelerated]
+= Deploy an accelerated workload
+:info_titleabbrev: Deploy accelerated workload
+
+include::../attributes.txt[]
+
+
+This tutorial demonstrates how Amazon EKS Auto Mode simplifies launching GPU-based workloads. Amazon EKS Auto Mode streamlines operations beyond the cluster itself by automating key infrastructure components providing compute, networking, load balancing, storage, and Identity Access and Management capabilities out of the box. 
+
+Amazon EKS Auto Mode includes the drivers required for certain instance types, such as NVIDIA and {aws} Neuron drivers. You do not have to install or update these drivers. 
+
+EKS Auto Mode manages the drivers for: 
+
+* link:ai/machine-learning/trainium/[{aws} Trainium, type="marketing"]
+* link:ai/machine-learning/inferentia/[{aws} Inferentia, type="marketing"]
+* NVIDIA GPU link:ec2/latest/instancetypes/ac.html[Amazon EC2 accelerated instances,type="documentation"]
+** Including the https://github.com/NVIDIA/k8s-device-plugin[NVIDIA device plugin for Kubernetes]. EKS Auto Mode includes this plugin, and is not visible as a daemon set within your cluster. 
+
+Amazon EKS Auto Mode also includes support for link:hpc/efa/[Elastic Fabric Adapter,type="marketing"].
+
+//GDC: What does the EFA thing mean?
```

Comment:
its a high speed, high throughput, low latency network used within HPC and machine learning applications where the workload spans multiple nodes (too large to fit on a single node). Its Amazon's version of Infiniband